bash/run-commands-in-parallel.md

```diff
@@ -0,0 +1,15 @@
+# Run Commands in Parallel
+I knew that running a command with a `&` after it sets it to a background task.
+However, I didn't think to run two commands that way...UNTIL NOW
+
+For example I need to run a bower install and a npm install
+```
+bower i & npm i
+```
+Neither task is dependent on each other and so doesn't require a `&&`. Under the hood
+the `&` forks a new process for that command. You can see the pid outputted
+when you run the command. To test:
+```
+echo "yo" & echo "what"
```

Comment:
a more pointant example might be `echo "yo" && sleep(10) & echo "what"` since `echo "yo" & echo "what"` will feel the same as `echo "yo" && echo "what"` ..right?